.storybook/manager.ts

```diff
@@ -0,0 +1,17 @@
+import { addons } from '@storybook/manager-api';
+import { create } from '@storybook/theming';
+
+const theme = create({
+  base: 'light',
+  // brandImage: logo,
```

Comment:
is this needed?

Reply:
is future reference for the logo of the tool